mRemoteV1/Connection/AbstractConnectionInfoData.cs

```diff
@@ -226,6 +227,15 @@ public ProtocolRDP.AuthenticationLevel RDPAuthenticationLevel
         }
 
         [LocalizedAttributes.LocalizedCategory("strCategoryProtocol", 3),
+            LocalizedAttributes.LocalizedDisplayName("strPropertyNameMinutesToIdleTimeout"),
+            LocalizedAttributes.LocalizedDescription("strPropertyDescriptionMinutesToIdleTimeout")]
```

Comment:
These localizations need to be changed to match the new variable names that were changed in the last commit.
line 230 should use `strPropertyNameRDPMinutesToIdleTimeout`
line 231 should use `strPropertyDescriptionRDPMinutesToIdleTimeout`

Change these two and I think we will be good.